ckcp/test.sh

```diff
@@ -11,18 +12,24 @@ KUBECONFIG_KCP="${KUBECONFIG_KCP:-$SCRIPT_DIR/work/kubeconfig/admin.kubeconfig}"
 get_namespace() {
   # Retrieve the KCP namespace id
   local ns_locator
-  ns_locator="{\"logical-cluster\":\"$(
+  ns_locator="\"workspace\":\"$(
     KUBECONFIG="$KUBECONFIG_KCP" kubectl kcp workspace current | cut -d\" -f2
```

Comment:
> KUBECONFIG_KCP="${KUBECONFIG_KCP:-$SCRIPT_DIR/work/kubeconfig/admin.kubeconfig}"

I think we'll need to change `$KUBECONFIG_KCP` to point to the kcp kubeconfig inside `$WORK_DIR/credentials/kubeconfig/kcp/-`, now generated every-time `openshift_dev_setup.sh` is run.

Reply:
There should be a way to pass the work dir as an argument when running the script.
I think that the whole script should be refactored to have a similar structure to the other scripts. That's something for another PR.

Reply:
I think I kinda missed your point. I've fixed the default path.